synapse/events/spamcheck.py

```diff
@@ -78,12 +80,14 @@ def user_may_invite(
         Returns:
             True if the user may send an invite, otherwise False
         """
-        if self.spam_checker is None:
-            return True
+        for spam_checker in self.spam_checkers:
+            if (
+                spam_checker.user_may_invite(inviter_userid, invitee_userid, room_id)
+                is False
+            ):
```

Comment:
Curious why this isn't just `if not spam_checker.user_may_invite(...)`?

Reply:
It was to ensure that if someone returned `None` from their module instead it would fail loudly (in that, it wouldn't work).

Reply:
Hmm...this seems like a change in behavior from before where we just pass this back through. I don't think it is a huge deal though.